src/extension/features/accounts/change-memo-enter-behavior/index.js

```diff
@@ -0,0 +1,38 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { isCurrentRouteAccountsPage } from 'toolkit/extension/utils/ynab';
+
+export class ChangeMemoEnterBehavior extends Feature {
+  shouldInvoke() {
+    return (
+      isCurrentRouteAccountsPage() &&
+      $('.ynab-grid-body-row.is-adding, .ynab-grid-body-row.is-editing').length
+    );
+  }
+
+  invoke() {
+    const $addEditRow = $('.ynab-grid-body-row.is-adding, .ynab-grid-body-row.is-editing');
+    const $memoInput = $('.ynab-grid-cell-memo input', $addEditRow);
+
+    if (!$memoInput[0].getAttribute('data-toolkit-memo-behavior')) {
+      $memoInput[0].setAttribute('data-toolkit-memo-behavior', true);
+      $memoInput.keydown(this.applyNewEnterBehavior);
+    }
+  }
+
+  applyNewEnterBehavior(event) {
+    if (event.keyCode === 13) {
+      event.preventDefault();
+      event.stopPropagation();
+
+      $('.ynab-grid-cell-outflow input').focus();
```

Comment:
What if the user has the `Add Check Number Column` feature turned on? For that reason you'd be better off using the array of columns instead of a named column.

Reply:
Updated to focus on the next column, regardless of what that is.